pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -213,61 +215,40 @@ func managePod(client coreclientv1.ConfigMapsGetter, recorder events.Recorder, o
 }
 
 func manageClientCABundle(lister corev1listers.ConfigMapLister, client coreclientv1.ConfigMapsGetter, recorder events.Recorder) (*corev1.ConfigMap, bool, error) {
-	inputs := []resourcesynccontroller.ResourceLocation{
+	requiredConfigMap, err := resourcesynccontroller.CombineCABundleConfigMaps(
+		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.TargetNamespace, Name: "client-ca"},
+		lister,
+		nil, // TODO remove this
+		nil, // TODO remove this
 		// this is from the installer and contains the value they think we should have
-		{Namespace: operatorclient.OperatorNamespace, Name: "initial-client-ca"},
+		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.GlobalUserSpecifiedConfigNamespace, Name: "initial-client-ca"},
```

Comment:
these namespaces changed when I eliminate the extra copy because of the combined lister we now have